Cargo.toml

```diff
@@ -233,21 +249,10 @@ rpath = false
 # If there are dependencies that need patching, they can be listed below.
 
 arrow2 = { git = "https://github.com/jorgecarleitao/arrow2", rev = "dd80c89" }
-arrow-format = { git = "https://github.com/sundy-li/arrow-format", rev = "c8e11341" }
+arrow-format = { git = "https://github.com/everpcpc/arrow-format", rev = "588d371" }
```

Comment:
do you pick the latest commit of `https://github.com/sundy-li/arrow-format` ?

Reply:
I cherry-picked some commits from it.